### PR TITLE
Updating README with new features

### LINKED ISSUE #22: Add `-delete` example to the README

### LINKED ISSUE #27: Add `-include` example in README

### PR DESCRIPTION
**Pull request by BrandonRomano**
Fixes #27 
Fixes #22

### CODE REVIEW

**Comment by BrandonRomano** [comment]
@kylemac here's a high level of all the new features that were added in the last few days

File: README.md [hunk 1]
```diff
@@ -51,6 +50,48 @@ blocked:000000
...
+#### Deleting Labels
+
+Sometimes you might want to delete some labels (likely Github's default labels) in the process of setting up a repo with your label config.
+
+You can add in the your label config file a line with `-delete:tag-name` to go and delete a specific label.
+
+For example:
+
+```
+-delete:wontfix
```
Comment (kylemac) [inline README.md:62]: it would be great to have a convenience option of something like `-delete:gh-default-labels`

Comment (BrandonRomano) [inline README.md:62]: We could add a custom label config that is only responsible for clearing gh labels.

`clear-default-labels`

```
-delete:wontfix
-delete:question
...
```

Then from a labels config, we could call:

```
-import:clear-default-labels
```

While this I imagine is a very popular use case, I would rather not update the code to handle a specific case like this.

**Comment by antonelli** [comment]
I like the `clear-default-labels` idea.

**Comment by antonelli** [comment]
:+1: to the readme

**Comment by BrandonRomano** [comment]
@antonelli I will create an issue for it.  @kylemac if you prefer your solution vs what I suggested we can have a discussion in the issue itself!

**Pull request merged**